iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/NetworkSerDe.java

```diff
@@ -874,14 +874,41 @@ public static Network copy(Network network, NetworkFactory networkFactory) {
     }
 
     public static Network copy(Network network, NetworkFactory networkFactory, ExecutorService executor) {
+        return copy(network, networkFactory, executor, TreeDataFormat.XML);
```

Comment:
Binary's still a beta feature, but shouldn't we default to JSON? any opinion @rolnico @geofjamg?

Reply:
JSON by default seems to be a good idea.